data/json/items/armor/eyewear.json

```diff
@@ -176,7 +176,7 @@
     "coverage": 20,
     "material_thickness": 1,
     "environmental_protection": 1,
-    "flags": [ "SUPER_FANCY", "FIX_NEARSIGHT", "FRAGILE" ]
+    "flags": [ "FANCY", "FIX_NEARSIGHT", "FRAGILE" ]
```

Comment:
What's the logic on this one not being SUPER_FANCY?

Reply:
Mostly because it's ridiculously out of style, same as the tricorne hat.